packages/terminal/src/browser/terminal-frontend-module.ts

```diff
@@ -54,7 +54,7 @@ export default new ContainerModule(bind => {
             const counter = terminalNum++;
             const domId = options.id || 'terminal-' + counter;
             const widgetOptions: TerminalWidgetOptions = {
-                title: 'Terminal ' + counter,
+                title: domId,
```

Comment:
@lmcbout why is the following change necessary?

Reply:
This prevent from having two terminals with the same number . If a terminal was created with the "id" = 3, it creates :terminal-3 and if the task creates a terminal with the counter =3, it would also creates a terminal: terminal-3  and it starts causing the issue. When we create a terminal from a task, we use "task-<counter> and it will never interfere with the terminal-<counter>

Reply:
@lmcbout I may be mistaken but here we are updating the terminal's title not its `id`.
Also, `options.id` used by `domId` is not guaranteed to be unique either so I don't see the reasoning behind updating the title here.

Reply:
We can use terminal-<counter> for task and terminal, at the end the title will be the "task:...> for the task . Since we do not restart the task after a back-end restart, it should not interfere anymore, but I prefer having the "task-<counter>" being defined for the task and "terminal-<counter>" for a pure terminal

Reply:
I still don't understand why updating the `title` here is necessary. 
Doesn't prefixing the terminal created when attaching tasks with `task-` sufficient?